cdc/sorter/leveldb/message/task.go

```diff
@@ -0,0 +1,55 @@
+// Copyright 2021 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package message
+
+import (
+	"fmt"
+
+	"github.com/pingcap/ticdc/cdc/sorter/encoding"
+	Iterator "github.com/syndtr/goleveldb/leveldb/iterator"
+	lutil "github.com/syndtr/goleveldb/leveldb/util"
+	"golang.org/x/sync/semaphore"
+)
+
+// Task is a leveldb actor task. It carries write and read request.
+type Task struct {
+	UID     uint32
+	TableID uint64
+
+	// encodeKey(tableID, event) -> serde.marshal(event)
+	// If a value is empty, it deletes the key/value entry in leveldb.
+	Events map[Key][]byte
```

Comment:
Is there an advantage of using `map` instead of a slice of structs?

Reply:
It's an analog of leveldb, key-value map. And it's easy to use.